app/clients/cloudwatch/aws_cloudwatch.py

```diff
@@ -116,19 +117,15 @@ def check_sms(self, message_id, notification_id, created_at):
         log_group_name = (
             f"sns/{region}/{account_number}/DirectPublishToPhoneNumber/Failure"
         )
-        current_app.logger.info(f"FAILURE LOG GROUP NAME {log_group_name}")
+        # current_app.logger.info(f"Failure log group name: {log_group_name}")
```

Comment:
If keeping this logging statement would be helpful in troubleshooting future issues, let's keep it uncommented.